src/category_theory/abelian/opposite.lean

```diff
@@ -29,4 +31,52 @@ instance : abelian Cᵒᵖ :=
   normal_epi := λ X Y f m, by exactI
     normal_epi_of_normal_mono_unop _ (abelian.normal_mono f.unop), }
 
+section
+
+variables {X Y : C} (f : X ⟶ Y)
+
+-- TODO: Generalize (this will work whenever f has a cokernel)
+-- (The abelian case is probably sufficient for most applications.)
+/-- The kernel of `f.op` is the opposite of `cokernel f`. -/
+@[simps]
+def kernel_op_unop : (kernel f.op).unop ≅ cokernel f :=
+{ hom := (kernel.lift f.op (cokernel.π f).op $ by simp [← op_comp]).unop,
+  inv := cokernel.desc f (kernel.ι f.op).unop $
+    by { erw [← f.unop_op, ← unop_comp, f.unop_op], simp },
```

Comment:
I'm increasingly hesitant about uses of `erw`, is there a specific reason it's needed here? Are there `dsimp` or `rw` lemmas we could add to avoid this?

Reply:
It works with `rw`. Fixed!